### ISSUE TITLE
Will this be shutting down once Uint8Array to/from base64 lands?

### ISSUE THREAD

**Issue by nektro** [feross/base64-js#2]
https://github.com/tc39/proposal-arraybuffer-base64 is currently at stage 3

**Comment by jonkoops** [comment]
No, I think `base64-js` would still serve a purpose for those that run in an environment that doesn't support this API. But I would certainly support adding a message to the README when things have stabilized and educate users of this option. We'd likely even call this API inside of this package if it is available in the environment of the consumer.
